desktop/main.js

```diff
@@ -184,6 +169,61 @@ const electronUpdater = browserWindow => ({
     },
 });
 
+/*
+* @param {Menu} systemMenu
+*/
+const localizeMenuItems = (browserWindow, systemMenu) => {
+    // List the Expensify Chat instance under the Window menu, even when it's hidden
+    systemMenu.insert(4, new MenuItem({
+        id: `historyMenuItem-${preferredLocale}`,
+        label: Localize.translate(preferredLocale, 'systemContextMenu.history'),
+        submenu: [{
+            id: `backMenuItem-${preferredLocale}`,
+            label: Localize.translate(preferredLocale, 'historyMenu.back'),
+            accelerator: process.platform === 'darwin' ? 'Cmd+[' : 'Shift+[',
+            click: () => { browserWindow.webContents.goBack(); },
+        },
+        {
+            id: `forwardMenuItem-${preferredLocale}`,
+            label: Localize.translate(preferredLocale, 'historyMenu.forward'),
+            accelerator: process.platform === 'darwin' ? 'Cmd+]' : 'Shift+]',
+            click: () => { browserWindow.webContents.goForward(); },
+        }],
+    }));
+
+    // Defines the system-level menu item to manually apply an update
+    // This menu item should become visible after an update is downloaded and ready to be applied
+    const updateAppMenuItem = new MenuItem({
+        id: `updateAppMenuItem-${preferredLocale}`,
+        label: Localize.translate(preferredLocale, 'systemContextMenu.updateExpensify'),
+        visible: false,
+        click: quitAndInstallWithUpdate,
+    });
+
+    // System-level menu item to manually check for App updates
+    const checkForUpdateMenuItem = new MenuItem({
+        id: `checkForUpdateMenuItem-${preferredLocale}`,
+        label: Localize.translate(preferredLocale, 'systemContextMenu.checkForUpdates'),
+        visible: true,
+        click: manuallyCheckForUpdates,
+    });
+
+    // Defines the system-level menu item for opening keyboard shortcuts modal
+    const keyboardShortcutsMenuItem = new MenuItem({
+        id: `keyboardShortcutsMenuItem-${preferredLocale}`,
+        label: Localize.translate(preferredLocale, 'initialSettingsPage.aboutPage.viewKeyboardShortcuts'),
+        accelerator: 'CmdOrCtrl+I',
+        click: () => {
+            showKeyboardShortcutsModal(browserWindow);
+        },
+    });
+
+    const appMenu = _.find(systemMenu.items, item => item.role === 'appmenu');
+    appMenu.submenu.insert(1, updateAppMenuItem);
```

Comment:
Is it correct to hard-code the indexes here?

Reply:
They were hardcoded before, I think it's harmless in the sense that it doesn't replace them, they just place them in that position and shift everything else.